packages/openapi/resources/mapi/datasources/main.yaml

```
@@ -14,6 +14,16 @@ paths:
       summary: Retrieve Multiple Datasources
       parameters:
         - $ref: ../shared/parameters.yaml#/space_id
+        - $ref: ../shared/pagination.yaml#/page
+        - name: per_page
+          in: query
+          required: false
+          schema:
+            type: integer
+            minimum: 1
+            maximum: 100
+            default: 25
+          description: Number of datasource per page. Default is 25, maximum is 100.
```

```suggestion
- $ref: ../shared/pagination.yaml#/per_page
```

Refactor suggestion: we can just import the per_page from the shared ones like this (but we need to add the `maximum: 100` in the shared one). In that way, we can also remove it from the other few places where is re-defined unnecessarily (except in `datasource_entries/main.yaml`, where `maximum: 1000` and in that case it needs to be redefined)